fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -1825,7 +1828,7 @@ ACTOR Future<Void> dataDistributionRelocator(DDQueue* self,
 			for (auto& destTeam : destinationTeams) {
 				totalIds += destTeam.servers.size();
 			}
-			if (totalIds != self->teamSize) {
```

Comment:
unfortunately it does not seem possible without a lot more work. I actually do not think this is the correct location for this warning anyways because the queue is not responsible for choosing the team